fineract-provider/build.gradle

```diff
@@ -81,8 +81,8 @@ dependencyManagement {
         dependency 'org.springframework.security.oauth:spring-security-oauth2:2.3.6.RELEASE'
         dependency "org.apache.openjpa:openjpa:3.1.1" // when upgrading, also change OpenJPA version repeated above in buildscript!
         dependency 'com.squareup.retrofit:retrofit:1.9.0'
-        dependency 'com.squareup.okhttp:okhttp:2.0.0'
-        dependency 'com.squareup.okhttp:okhttp-urlconnection:2.0.0'
+        dependency 'com.squareup.okhttp:okhttp:+'
+        dependency 'com.squareup.okhttp:okhttp-urlconnection:+'
```

Comment:
As per comment in line 79 of this file, we actually do NOT want to use '+' versions - because they're "unpredictable". Instead, manually fix the latest one currently available here.